exampleSite/config.toml

```diff
@@ -94,7 +94,7 @@ title = "Gokarna"
 
 [markup]
   [markup.tableOfContents]
-    startLevel = 1
+    startLevel = 2
```

Comment:
toc on the [deploy preview](https://deploy-preview-267--gokarna-hugo.netlify.app/posts/markdown-syntax/) doesn't show all the headings anymore. See "Headings" and "H1" are missing on the left side toc. I'm guessing it's due to this change:

<img width="1698" alt="image" src="https://github.com/user-attachments/assets/2a7f4eea-f949-4cd5-ab2a-ff89ccd39157">

Reply:
@526avijitgupta Reverted the offending commits :+1: